packages/stream-server/package.js

```diff
@@ -0,0 +1,20 @@
+Package.describe({
+  summary: "Meteor's DDP stream server based on SockJS",
+  version: '2.2.0',
+  documentation: 'README.md'
+});
+
+Npm.depends({
+  "permessage-deflate": "0.1.6",
+  sockjs: "0.3.19"
+});
+
+Package.onUse(function (api) {
+  api.use(['underscore'], 'server');
```

Comment:
We've been trying to remove `underscore` from core packages for a while now. Let's avoid it here if possible.

Reply:
@benjamn I has copy/paste code of old `stream_server.js` to `stream-server` package.

I can rewrite it to the full ES2015+ support if you need so.